core/commands/repo.go

```diff
@@ -19,10 +21,17 @@ var RepoCmd = &cmds.Command{
 	},
 
 	Subcommands: map[string]*cmds.Command{
-		"gc": repoGcCmd,
+		"gc":   repoGcCmd,
+		"stat": repoStatCmd,
 	},
 }
 
+type RepoStat struct {
+	localPath string
+	szBytes   uint64
+	szBlocks  uint64
```

Comment:
Yeah, numBlocks would be better since BlockSize isn't constant. How does

	type RepoStat struct {
		repoPath  string
		repoSize  uint64 // [B]
		numBlocks uint64
	}

look?